gslib/commands/acl.py

```diff
@@ -339,6 +342,45 @@ class AclCommand(Command):
           'ch': _ch_help_text
       },
   )
+  CannedGcsAcl = ['private', 'public-read', 'project-private',
+                    'public-read-write', 'authenticated-read',
+                    'bucket-owner-read', 'bucket-owner-full-control']
+  def get_gcloud_storage_args(self):
+    if self.args[0] == 'get':
+      url = self.args[1]
+      object_or_bucket = StorageUrlFromString(url)
+      if object_or_bucket.IsObject():
+        command_group = 'objects'
+      else:
+        command_group = 'buckets'
+      gcloud_storage_map = GcloudStorageMap(gcloud_command=[
+            'alpha', 'storage', command_group, 'describe', url, '--format=json[acl]'
+        ],
+                                              flag_map={})
+   
+    elif self.args[0] == 'set':
+      url = self.args[2]
```

Comment:
There can be more than one URL.
https://cloud.google.com/storage/docs/gsutil/commands/acl